Tests/KituraTests/KituraTest.swift

```diff
@@ -132,17 +142,21 @@ class KituraTest: XCTestCase {
         }
 
         do {
+#if USE_EPHEMERAL_PORTS
```

Comment:
Is there any benefit to having this as an option - why not just use ephemeral ports all the time? We could just have one test that tests we can listen on a specific port, for coverage.

Reply:
Again, this is keep the local testing process undisturbed. The `sysctl` commands need superuser access (at least on `macOS`). If we are willing to provide instructions to run the tests locally, we could do away with the compiler flag.

Reply:
Do the Kitura tests exhaust the ephemeral port range if we don't do this? By default, ports 49152 - 65535 are available (16k), so I'd be surprised if this wasn't sufficient for a single execution of the tests.